### PR TITLE
whitespace(-mostly) style cleanup.

### PR DESCRIPTION
**Pull request by anntzer**
I intentionally left in nearly all commented out dead code.

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/mpl_toolkits/axisartist/floating_axes.py [hunk 1]
```diff
@@ -161,15 +153,14 @@ def transform_xy(x, y):
...
-            mm = ((yy1b-yy1a)==0.) & ((xx1b-xx1a)==0.) # mask where dd1 is not defined
+            dd = np.arctan2(yy1b-yy1a, xx1b-xx1a)  # angle normal
+            dd2 = np.arctan2(yy2b-yy2a, xx2b-xx2a)  # angle tangent
+            mm = ((yy1b-yy1a)==0.) & ((xx1b-xx1a)==0.)  # mask where dd1 is not defined
             dd[mm] = dd2[mm] + np.pi / 2
 
-            #dd += np.pi
-            #dd = np.arctan2(xx2-xx1, angle_tangent-yy1)
+            # dd += np.pi
+            # dd = np.arctan2(xx2-xx1, angle_tangent-yy1)
```
Comment (phobson) [inline lib/mpl_toolkits/axisartist/floating_axes.py]: should these just go away?

File: lib/mpl_toolkits/axisartist/floating_axes.py [hunk 2]
```diff
@@ -245,30 +235,21 @@ def __init__(self, aux_trans, extremes,
                          tick_formatter1=tick_formatter1,
                          tick_formatter2=tick_formatter2)
 
-
```
Comment (phobson) [inline lib/mpl_toolkits/axisartist/floating_axes.py]: should these go away as well?

File: lib/mpl_toolkits/axisartist/floating_axes.py [hunk 3]
```diff
@@ -374,23 +350,18 @@ def _update_grid(self, x1, y1, x2, y2):
...
         grid_info["lon_lines"] = lon_lines
         grid_info["lat_lines"] = lat_lines
 
-
         lon_lines, lat_lines = grid_finder._get_raw_grid_lines(extremes[:2],
                                                                extremes[2:],
                                                                *extremes)
-        #lon_min, lon_max,
+        # lon_min, lon_max,
         #                                                       lat_min, lat_max)
```
Comment (phobson) [inline lib/mpl_toolkits/axisartist/floating_axes.py]: perhaps this should be bumped up to previous line

File: lib/mpl_toolkits/axisartist/grid_helper_curvelinear.py [hunk 4]
```diff
@@ -140,7 +138,7 @@ def update_lim(self, axes):
 
         grid_finder = self.grid_helper.grid_finder
 
-        #e1, e2 = self._extremes # ranges of other coordinates
+        # e1, e2 = self._extremes # ranges of other coordinates
```
Comment (phobson) [inline lib/mpl_toolkits/axisartist/grid_helper_curvelinear.py]: goes away?

File: lib/mpl_toolkits/axisartist/grid_helper_curvelinear.py [hunk 5]
```diff
@@ -272,20 +267,19 @@ def transform_xy(x, y):
...
+            mm = ((yy1b-yy1a)==0.) & ((xx1b-xx1a)==0.)  # mask where dd1 is not defined
             dd[mm] = dd2[mm] + np.pi / 2
-            #dd = np.arctan2(yy2-yy1, xx2-xx1) # angle normal
-            #dd2 = np.arctan2(yy3-yy1, xx3-xx1) # angle tangent
-            #mm = ((yy2-yy1)==0.) & ((xx2-xx1)==0.) # mask where dd1 is not defined
-            #dd[mm] = dd2[mm] + np.pi / 2
+            # dd = np.arctan2(yy2-yy1, xx2-xx1) # angle normal
+            # dd2 = np.arctan2(yy3-yy1, xx3-xx1) # angle tangent
+            # mm = ((yy2-yy1)==0.) & ((xx2-xx1)==0.) # mask where dd1 is not defined
+            # dd[mm] = dd2[mm] + np.pi / 2
```
Comment (phobson) [inline lib/mpl_toolkits/axisartist/grid_helper_curvelinear.py]: goes away?

**Comment by anntzer** [comment]
I agree with removing dead code in general, but I think someone should first figure out what a given line was intended to do before killing it.

File: lib/mpl_toolkits/axisartist/floating_axes.py (hunk 1 above)
Comment (anntzer) [reply lib/mpl_toolkits/axisartist/floating_axes.py]: The whole block came in commented out with the surrounding code in 4fa9271 so yes.

File: lib/mpl_toolkits/axisartist/floating_axes.py (hunk 2 above)
Comment (anntzer) [reply lib/mpl_toolkits/axisartist/floating_axes.py]: there's an update_grid_finder in grid_helper_curvelinear so I'll leave this one in

**Comment by WeatherGod** [comment]
I really don't like it when auto-linters add that space for blocks of
commented out code. It doesn't make it any easier for me to read, and it
actually makes it harder to distinguish an expression that lives in a
comment, versus some actual, previously used but now commented out code.

File: lib/mpl_toolkits/axisartist/grid_helper_curvelinear.py (hunk 4 above)
Comment (anntzer) [reply lib/mpl_toolkits/axisartist/grid_helper_curvelinear.py]: duplicated just a bit before in the same function so yes

File: lib/mpl_toolkits/axisartist/grid_helper_curvelinear.py (hunk 5 above)
Comment (anntzer) [reply lib/mpl_toolkits/axisartist/grid_helper_curvelinear.py]: came in commented in 3b45ebbg, and there's no yy2 and friends as local variables anymore, so I guess sure.

**Comment by anntzer** [comment]
I hadn't heard of the "if there's no space after the hash it's commented out code" convention; I guess it could be a valuable one but we're already far from doing that in the rest of the codebase (yes, this PR makes things "worse" in that respect).
(Also that's basically arguing "we don't care about this specific item of the style guide", which is fine but another discussion...)

File: lib/mpl_toolkits/axisartist/angle_helper.py [hunk 6]
```diff
@@ -343,14 +342,13 @@ def __init__(self,
                  lon_minmax = None,
                  lat_minmax = (-90, 90)
                  ):
-        #self.transform_xy = transform_xy
-        #self.inv_transform_xy = inv_transform_xy
+        # self.transform_xy = transform_xy
+        # self.inv_transform_xy = inv_transform_xy
```
Comment (timhoffm) [inline lib/mpl_toolkits/axisartist/angle_helper.py]: Stale commented code. Should be removed.

File: lib/mpl_toolkits/axisartist/axislines.py [hunk 7]
```diff
@@ -242,12 +242,12 @@ def get_tick_iterators(self, axes):
...
                 for x, l in zip(locs, labels):
 
-                    c = list(self.passthru_pt) # copy
+                    c = list(self.passthru_pt)  # copy
                     c[self.nth_coord] = x
 
                     # check if the tick point is inside axes
                     c2 = tr2ax.transform_point(c)
-                    #delta=0.00001
+                    # delta=0.00001
```
Comment (timhoffm) [inline lib/mpl_toolkits/axisartist/axislines.py]: Stale code comment?

File: lib/mpl_toolkits/axisartist/floating_axes.py [hunk 8]
```diff
@@ -26,9 +26,9 @@ def __init__(self, grid_helper, side, nth_coord_ticks=None):
          nth_coord = 0 ->  x axis, nth_coord = 1 -> y axis
         """
 
-        value, nth_coord = grid_helper.get_data_boundary(side) # return v= 0 , nth=1, extremes of the other coordinate.
+        value, nth_coord = grid_helper.get_data_boundary(side)  # return v= 0 , nth=1, extremes of the other coordinate.
         super().__init__(grid_helper, nth_coord, value, axis_direction=side)
-        #self.grid_helper = grid_helper
+        # self.grid_helper = grid_helper
```
Comment (timhoffm) [inline lib/mpl_toolkits/axisartist/floating_axes.py]: Stale code comment. Remove.

File: lib/mpl_toolkits/axisartist/floating_axes.py [hunk 9]
```diff
@@ -123,7 +116,7 @@ def transform_xy(x, y):
             xx0 = np.empty_like(yy0)
             xx0.fill(self.value)
 
-            #yy0_ = yy0.copy()
+            # yy0_ = yy0.copy()
```
Comment (timhoffm) [inline lib/mpl_toolkits/axisartist/floating_axes.py]: Stale code comment. Remove.

File: lib/mpl_toolkits/axisartist/floating_axes.py [hunk 10]
```diff
@@ -144,10 +137,9 @@ def transform_xy(x, y):
             yy0 = np.empty_like(xx0)
             yy0.fill(self.value)
 
-            #xx0_ = xx0.copy()
+            # xx0_ = xx0.copy()
```
Comment (timhoffm) [inline lib/mpl_toolkits/axisartist/floating_axes.py]: Stale code comment. Remove.

File: lib/mpl_toolkits/axisartist/floating_axes.py [hunk 11]
```diff
@@ -485,10 +439,9 @@ def cla(self):
 
         self._original_patch = patch
 
-
     def adjust_axes_lim(self):
 
-        #t = self.get_boundary()
+        # t = self.get_boundary()
```
Comment (timhoffm) [inline lib/mpl_toolkits/axisartist/floating_axes.py]: Stale code comment. Remove.

**Comment by anntzer** [comment]
All cases you pointed out look reasonable (except perhaps FixedLocator(nbins=10) because who knows whether there was a real need to set nbins=10, OTOH that just seems unlikely), removed them all.

**Pull request merged**